web/js/main.js

```diff
@@ -26,6 +26,23 @@
                 close: 'fa fa-remove'
             }
         });
+
+        // Bootstrap-tagsinput initialization
+        // http://bootstrap-tagsinput.github.io/bootstrap-tagsinput/examples/
+        var $input = $('input[data-toggle="tagsinput"]');
+        var source = new Bloodhound({
```

Comment:
this should be done only when ``$input.length`` is not 0 (i.e. at least one such input has been found on the page)